packages/components/src/index.js

```diff
@@ -25,12 +25,7 @@ export { default as BaseControl } from './base-control';
 export { default as __experimentalBoxControl } from './box-control';
 export { default as Button } from './button';
 export { default as ButtonGroup } from './button-group';
-export { default as Card } from './card';
-export { default as CardBody } from './card/body';
-export { default as CardDivider } from './card/divider';
-export { default as CardFooter } from './card/footer';
-export { default as CardHeader } from './card/header';
-export { default as CardMedia } from './card/media';
+export { Card, CardBody, CardDivider, CardFooter, CardHeader, CardMedia } from './card';
```

Comment:
Seems like those need to go on separate lines - the linter complains 😉

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/32557/commits/564c683c2a3ff5321d39265075555b995f0746e5